### DISCUSSION TITLE
Blog posts on Flet

### DISCUSSION THREAD

**Discussion by ndonkoHenri** [flet-dev/flet#643] [Show and tell]
I started a personal and a medium blog. For now, I will be writing on Flet stuffs.

Have a look at my first post on [building a Markdown editor with flet](https://ndonkohenri.medium.com/building-a-markdown-editor-previewer-with-flet-7d9b06d6dc4b?source=user_profile---------0----------------------------) and let me know eventually what you think of it..😉

**Comment by ndonkoHenri** [comment]
I pushed a second blog post on [how to build a URL shortener app with Flet](https://ndonkohenri.medium.com/building-a-url-shortener-flutter-app-with-flet-python-framework-fffa1d98a53e).
Let me know what you think of it, please.

**Reply by FeodorFitsner** [reply]
If you like you can do a pull request to tutorials section on Flet website: https://flet.dev/docs/tutorials It would be awesome to have more content there!

**Comment by ndonkoHenri** [comment]
Posted a new article some days ago on **How to resize Flet’s TextField control without the content going invisible**
https://ndonkohenri.medium.com/how-to-resize-flets-textfield-control-without-the-content-going-invisible-6d268a4d96e1

**Comment by ndonkoHenri** [comment]
New article posted some days ago:
https://ndonkohenri.medium.com/building-a-random-image-generator-flutter-app-with-the-flet-python-framework-ecfe8b5daaf8

**Comment by ndonkoHenri** [comment]
Two new articles.
**Rebuilding the Flutter Counter App in Python with Flet**: https://medium.com/@ndonkohenri/rebuilding-the-flutter-counter-app-in-python-with-flet-53d4a2e29a65

**Deploying a Flet App for Free on Cloudflare Pages**: https://ndonkohenri.medium.com/deploying-a-flet-app-for-free-on-cloudflare-pages-e56ecc6ce450

Let me have your _Fletic_ impressions.

**Reply by Johnynight** [reply]
When using your instructions on 08/14/2013 when deploying the application, an infinite download appears

**Reply by ndonkoHenri** [reply]
Hey! Please can you provide more context?

**Comment by ndonkoHenri** [comment]
New Post: **Deploying a Flet App on Replit**
https://ndonkohenri.medium.com/deploying-unlimited-python-flutter-apps-for-free-on-replit-flet-ceba88fbaea7
